src/api/client.hpp

```diff
@@ -79,14 +79,6 @@ class HTTPClient : public Client {
 		authenticator_.ExpireToken();
 	}
 
-	bool HasReauthenticated() {
-		return reauthenticated_;
-	}
-
-	void SetReauthenticated(bool reauthenticated) {
-		reauthenticated_ = reauthenticated;
```

Comment:
You should also remove the private `reauthenticated_` variable, as it's not used anymore

Reply:
Yes, I've done that locally but haven't submitted it yet. Saw the failing tests.